src/language/ast.ts

```diff
@@ -323,14 +323,18 @@ export interface OperationDefinitionNode {
   readonly selectionSet: SelectionSetNode;
 }
 
-export enum OperationTypeNode {
-  QUERY = 'query',
-  MUTATION = 'mutation',
-  SUBSCRIPTION = 'subscription',
-}
+export const OperationTypeNode = Object.freeze({
+  QUERY: 'query',
+  MUTATION: 'mutation',
+  SUBSCRIPTION: 'subscription',
+} as const);
+
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export type OperationTypeNode =
+  typeof OperationTypeNode[keyof typeof OperationTypeNode];
```

Comment:
Please extract it into `jsutils` under descriptive name, e.g. `ValueTypes`
totally open to bike shedding